### ISSUE TITLE
Mutiselect optionValue issue

### ISSUE THREAD

**Issue by vrinceanuv** [primefaces/primereact#1950]
**I'm submitting a ...** 
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://forum.primefaces.org/viewforum.php?f=57
```
**Codesandbox Case (Bug Reports)**
https://codesandbox.io/s/multi-select-optiovalue-bug-0ef66

**Current behavior**
When selecting a value the checkbox is not checked. It also allows you to select the same option over and over again.
<img width="645" alt="Screenshot 2021-04-12 at 13 46 54" src="https://user-images.githubusercontent.com/4588351/114382965-ab38f080-9b95-11eb-8a09-3aa286491524.png">

**Expected behavior**
when selecting a value the checkbox should be checked

**Minimal reproduction of the problem with instructions**
Go to the sandbox url and use the dropdown
<img width="419" alt="Screenshot 2021-04-12 at 13 33 38" src="https://user-images.githubusercontent.com/4588351/114381983-78422d00-9b94-11eb-831f-1a86f481aa68.png">

**Please tell us about your environment:**
not relevant

* **React version:**
17

* **PrimeReact version:**
6.2.1

* **Browser:** [all]
 
* **Language:** [ES6/7 | ES5]

**Comment by canpan14** [comment]
Related to #1927 and #1897

**Comment by mertsincan** [comment]
Hi,
Fixed in #1897

Best Regards,
